### PR TITLE
Update AB layout to match Aqua layout (ad spots) and create 'Daily Dig' section/ad 

### PR DESCRIPTION
**Pull request by jwade1327**
Screen Shot 2024-01-30 at 10 31 04 AM
Screen Shot 2024-01-30 at 10 31 24 AM

### CODE REVIEW

**Comment by jwade1327** [comment]
[AB: Enews ad look, Sponsored Content look change requests](https://trello.com/c/9IOruuu9/634-ab-enews-ad-look-sponsored-content-look-change-requests)

File: packages/common/components/blocks/ad/promotion-daily-dig.marko [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { get, getAsObject } from "@parameter1/base-cms-object-path";
+
+$ const newsletter = getAsObject(input, "newsletter");
+$ const { sectionName, date, content, advertiser } = input;
+
+$ content.labels = ["Sponsored"];
```
Comment (Shinsina) [inline packages/common/components/blocks/ad/promotion-daily-dig.marko:6]: This doesn't appear to be actively used by the daily-dig-block, is there a reason it's included here?

File: packages/common/components/blocks/content/daily-dig.marko [hunk 2]
```diff
@@ -0,0 +1,116 @@
+import { get, getAsArray } from "@parameter1/base-cms-object-path";
```
Comment (Shinsina) [inline packages/common/components/blocks/content/daily-dig.marko:1]: Due to ` $ const labels = getAsArray(content, "labels");` being the only thing using that and as far as I can tell `labels` goes unused here `getAsArray` can likely be removed here

File: packages/common/components/blocks/content/daily-dig.marko [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+$ const imagePosition = defaultValue(input.imagePosition, 'right');
+$ const withSection = defaultValue(input.withSection, true);
+$ const continueReading = defaultValue(input.continueReading, false);
+$ const withTeaser = defaultValue(input.withTeaser, true);
+
+$ const queryString = (urlParams && content.type !== 'promotion') ? `?${new URLSearchParams(urlParams)}` : "";
+$ const contentUrl = `${content.siteContext.url}${queryString}`
+
+$ const newsletterConfig = config.get(newsletter.alias);
+$ const primaryColor = defaultValue(get(newsletterConfig, "primaryColor"), "#103A57");
```
Comment (Shinsina) [inline packages/common/components/blocks/content/daily-dig.marko:19]: These both can be removed as this is the only thing using `newsletterConfig` and `primaryColor` isn't used anywhere

**Review by Shinsina — CHANGES_REQUESTED** [review]
If my understanding of what's actually going on here is incorrect do let me know.

**Review by Shinsina — APPROVED** [review]

**Pull request merged**